index.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
+    <link rel="stylesheet" href="./style.css">
+    <title>Samuel Kinyanjui</title>
+</head>
+<body>
+   <section class="myportfolio"> 
+
+        <section class="toolbar_headline flex-item ">
+            <a href="toolbar_headline"></a>
+            <header class="headerx">
+                <div class="logox"> <img src="./img/Mylogo.png" alt="Samuel Logo">
+                </div>
+
+                <nav class="navigation">
+                    <img src="./img/menu.png"> 
+                
+                </nav>
+            </header>      
+
+            <section class="headline">            
+                <div class="headerh1">
+                <h1>I’m Samuel k.
+                    Glad to see you!</h1>
+                </div>
+                <div class="headerdetails">
+                    <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
+                    
+                </div>
+                <div class="letconnects">
+                <h4>LET’S CONNECT</h4>  
+                </div>
+                <div class="socialmediaclass">
+                    <img id="socialmedia" src="./img/socialmedia.png">
+                </div>
+
+            </section>
+        </section>
+
+        <section class="Works" >
+            <div  class="workflex">
+                <div class="workgrid">
```

Comment:
- [ ] You are currently not making use of css grid property correctly in your project, the requirement says you are to add css grid to the overall container for the works section and style each work card using flex, but with your code base you have added grid to each work card using `workgrid` class selector rather you are to add the display grid to your container element fwhich is to hold all of the cards of your works section . It is important you add this styling to your code base becasue it will come in handy when you start building the desktop version of your application :+1: :sparkles: